evap/evaluation/templatetags/navbar_templatetags.py

```diff
@@ -14,4 +14,5 @@ def include_navbar(user, language):
         'published_result_semesters': Semester.get_all_with_published_unarchived_results(),
         'result_semesters': Semester.get_all_with_unarchived_results(),
         'grade_document_semesters': Semester.objects.filter(grade_documents_are_deleted=False),
+        'DEBUG': DEBUG,
```

Comment:
Hm, I’d prefer to keep template variables in lowercase; `languages` is also transformed even though it originates from settings